.github/workflows/docs-build.yaml

```diff
@@ -70,7 +75,7 @@ jobs:
         with:
           fetch-depth: 1
       - name: Log in to Container Registry
-        uses: docker/login-action@v2
+        uses: docker/login-action@v3
```

Comment:
It's not critical, but we may want to configure dependabot on this repo to automate updates such as these.